nmostesting/suites/IS0702Test.py

```diff
@@ -481,6 +485,222 @@ def test_05(self, test):
         else:
             return test.UNCLEAR("Not tested. No resources found.")
 
+    def test_06(self, test):
+        """MQTT senders on the same device have the same destination host/port
+            broker_protocol and broker_authorization parameters"""
+
+        if len(self.is07_sources) > 0:
+            found_senders = False
+            senders_by_device = {}
+            for source_id in self.is07_sources:
+                if source_id in self.sources_to_test:
+                    for sender_id in self.senders_to_test:
+                        flow_id = self.senders_to_test[sender_id]["flow_id"]
+                        if flow_id in self.is04_flows:
+                            if source_id == self.is04_flows[flow_id]["source_id"]:
+                                found_sender = self.senders_to_test[sender_id]
+                                if found_sender["transport"] == "urn:x-nmos:transport:mqtt":
+                                    if found_sender["device_id"] not in senders_by_device:
+                                        senders_dict = {}
+                                        senders_dict[found_sender["id"]] = found_sender
+                                        senders_by_device[found_sender["device_id"]] = senders_dict
+                                    else:
+                                        senders_dict = senders_by_device[found_sender["device_id"]]
+                                        senders_dict[found_sender["id"]] = found_sender
+
+            for device_id in senders_by_device:
+                device_destination_host = None
+                device_destination_port = None
+                device_broker_protocol = None
+                device_broker_authorization = None
+                senders_dict = senders_by_device[device_id]
+                for sender_id in senders_dict:
+                    found_sender = senders_dict[sender_id]
+                    if found_sender["id"] in self.senders_active:
+                        found_senders = True
+                        try:
+                            params = self.senders_active[found_sender["id"]]["transport_params"][0]
+                            sender_destination_host = params["destination_host"]
+                            sender_destination_port = params["destination_port"]
+                            sender_broker_protocol = params["broker_protocol"]
+                            sender_broker_authorization = params["broker_authorization"]
+
+                            if device_destination_host is None:
+                                device_destination_host = sender_destination_host
+                            else:
+                                if device_destination_host != sender_destination_host:
```

Comment:
The `destination_host` may be `null` (`None`), which indicates that the Sender has not yet been configured. Should this handle a device which has some configured Senders and some not?